src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -717,7 +719,17 @@ def list_connectedk8s(cmd, client, resource_group_name=None):
 
 
 def delete_connectedk8s(cmd, client, resource_group_name, cluster_name,
-                        kube_config=None, kube_context=None, no_wait=False):
+                        kube_config=None, kube_context=None, no_wait=False, force_delete=False, yes=False):
+
+    # The force delete prompt is added because it can be used in the case where the config map is missing
+    # so we cannot check if the user context is pointing to the cluster that he intends to delete
+    if not force_delete:
+        confirmation_message = "Are you sure you want to perform delete operation?"
+        utils.user_confirmation(confirmation_message, yes)
+    elif force_delete:
+        confirmation_message = "Force delete will clean up all the arc custom resources. Please make sure your current kubeconfig is pointing to the right cluster.\n" + "Are you sure you want to perform force delete operation:"
```

Comment:
@anraghun to review CX facing text

Reply:
Force delete will clean up all the ~arc~ azure-arc ~custom~ resources, including extensions. Please make sure your current kubeconfig is pointing to the right cluster.\n" + "Are you sure you want to perform force delete ~operation~: